test/integration/supervisor.spec.ts

```diff
@@ -0,0 +1,10 @@
+import * as request from 'supertest';
+
+const BALENA_SUPERVISOR_ADDRESS =
+	process.env.BALENA_SUPERVISOR_ADDRESS || 'http://balena-supervisor:48484';
+
+describe('supervisor app', () => {
```

Comment:
Could we consider removing `test/legacy/18-startup.spec.ts` since we have this?

Reply:
yep, good call

Reply:
done